python/paddle/distributed/auto_parallel/operators/common.py

```diff
@@ -114,3 +114,44 @@ def find_best_compatible_distributed_operator_impl(name, op_dist_attr,
         best_compatible_impl, idx = None, -1
 
     return best_compatible_impl, idx
+
+
+def set_distributed_attr_for_var(src_op_dist_attr, var, src_var):
```

Comment:
function naming could be more self-explained.
here you actually copied all the dist_attr info from src to dst var, it is more close to  a "copy" than a "setting" semantically. 
use something like "copy_distributed_attr_for_var" or check if a similar function has already been implemented in context.py

Reply:
done.